ViAn/GUI/mainwindow.cpp

```diff
@@ -947,6 +1105,48 @@ void MainWindow::remove_item_from_tree(MyQTreeWidgetItem *my_item) {
 }
 
 /**
+ * @brief MainWindow::remove_analysis_from_queue
```

Comment:
the functions "remove_analysis_of_video" and remove_analysis_of_project" had the description "remove analysis from the queue" which should be the description for this function according to the names.